commit ea01e14c39b90a04696ed91dffc2692650927469
Author: Seth Forshee <seth.forshee@canonical.com>
Date:   Fri Jun 2 13:45:22 2017 -0500

    UBUNTU: SAUCE: (efi-lockdown) efi: Don't print secure boot state from the efi stub
    
    During boot the efi stub prints what amounts to debugging
    messages about the secure boot state to the efi console. which
    appear on the screen during boot. The same information is printed
    in dmesg while the kernel is booting, so they serve no purpose
    aside from debugging issues in the efi stub. Remove them.
    
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/drivers/firmware/efi/libstub/secureboot.c b/drivers/firmware/efi/libstub/secureboot.c
index 959777ec8a77..dade32e658cb 100644
--- a/drivers/firmware/efi/libstub/secureboot.c
+++ b/drivers/firmware/efi/libstub/secureboot.c
@@ -77,10 +77,8 @@ enum efi_secureboot_mode efi_get_secureboot(efi_system_table_t *sys_table_arg)
 		return efi_secureboot_mode_disabled;
 
 secure_boot_enabled:
-	pr_efi(sys_table_arg, "UEFI Secure Boot is enabled.\n");
 	return efi_secureboot_mode_enabled;
 
 out_efi_err:
-	pr_efi_err(sys_table_arg, "Could not determine UEFI Secure Boot status.\n");
 	return efi_secureboot_mode_unknown;
 }
